hugo/layouts/features/single.html

```diff
@@ -0,0 +1,93 @@
+{{ $projects := $.Site.Data.projects }}
+
+{{ $headCtx := dict "ctx" . }}
+{{ partial "head.html" $headCtx }}
+
+<body>
+{{ partial "header.html" }}
+<div class="text-left pb-5">
+    <br><br>
+    <h1 class="title text-left p-0 mt-5 mb-4">source{d} integrates with you favorite Development tools & platforms</h1>
+</div>
+</div>
+</header>
+
+<section id="visualize" class="pt-5 pb-0">
+    <div class="container">
+        <header class="mb-5 text-center">
+            <h2 class="text-dark">Visualize your software development data  to make better decisions</h2>
+        </header>
+        <div class="row">
+            <div class="col-sm-6 mb-5">
+                <h3 class="text-medium-light mb-4">Transparent roadmap and Ecosystem collaboration</h3>
+                <p class="text-dark">With source{d} you can start getting answers to tough questions in a matter of minutes with simple SQL queries. source{d} supports all major operating systems (Linux, Windows, MacOS), hosting environments (Cloud or On-premise), version control systems (GitHub, GitLab, Bitbucket, etc) and adapts to different stages of development, workflows and business needs. Enjoy the flexibility to choose whether to work on our built-in interactive web interface, your favorite SQL client, command-line interface or Business Intelligence platform (Tableau, Looker, PowerBI, etc).</p>
+                <div class="text-center d-flex mt-5">
+                    <a href="#" class="btn btn-purple text-white flex-fill mr-4">Download source{d}</a>
+                    <a href="#" class="btn btn-highlight flex-fill">Request a Demo</a>
+                </div>
+            </div>
+            <div class="col-sm-6 mb-5">
+                <img class="img-fluid p-3" src="img/visualize.jpg" alt="">
+            </div>
+        </div>
+    </div>
+</section>
+
+<section id="all" class="py-5">
+    <div class="container text-dark">
+        <header class="text-center mb-5">
+            <h2>Get all your code and engineering data in one place</h2>
+        </header>
+        <div class="row">
+            <div class="col-sm-6 mb-5">
+                <img class="img-fluid p-3" src="img/all.svg" alt="">
```

Comment:
This diagram says `source{d} Engine`. I think it should be `source{d}`
![image](https://user-images.githubusercontent.com/2437584/58325838-536f3000-7e2b-11e9-98d7-e9b0d046b419.png)